packages/component-library/stories/horizontalBarChart.notes.md

```diff
@@ -28,3 +28,13 @@ The Custom story shows all possible properties that can be passed to the BarChar
 ## Example: Minimalist
 
 ## Example: With Negative Values
+
+## Example: Stacked Bar Chart
+
+The Stacked Bar Chart shows all the possible properties for the stacked option of the Horizontal Bar Chart.
+
+The Stacked Bar Chart requires the `stacked` and `groupByValue` props. The `groupByValue` is the value that you would like to data to be grouped by. This is the value that is represented by the color of the bar, not the value of the x or y axis. This value is needed because the chart requires an array of arrays of objects, and the standard data structure for CIVIC is an array of objects.
```

Comment:
```suggestion
The Stacked Bar Chart requires the `stacked` and `dataSeries` props. The `dataSeries` is the value that you would like to data to be grouped by. This is the value that is represented by the color of the bar, not the value of the x or y axis.
```

Reply:
Removed the last sentence since it's an implementation detail not relevant to the component user